resources/frontend-web/fundamental/README.md

```diff
@@ -0,0 +1,35 @@
+# Glosarium
+<div style="text-align: justify">
+Berfungsi agar bila kita ingin mencari suatu topik khusus, gak perlu buka semua resourcenya satu per satu. Berisi deskripsi singkat mengenai resource yang ada di folder ini. Bila ingin menkontribusi bacaan jangan lupa di buat ringkasannya dan di tambahkan di sini.
+
+1) HTML 😎
+\
+`HTML` adalah singkatan dari `Hyper Text Markup Language` yang merupakan standar dalam pembuatan halaman website
+<br>
+konten ini berisi terkait fundamental dari html seperti:
+    - definition,
+    - function,
+    - how to use,
+    - tutorial,
+    - etc.
+2) CSS 🤩
+\
+`CSS` merupakan singkatan dari `Cascading Style Sheet` yang berfungsi sebagai styling html tags, jadi style seperti warna, ukuran sebuah objek, font-family, dan semua hal yang berkaitan dengan desain sebuah web menggunakan `CSS`, tentunya banyak sekali manfaat serta tips & tricks dari `CSS` ini.
+<br>
+untuk mencari tahu lebih lanjut, kami telah menyediakan beberapa resource terkait css yang berisi:
+    - definition,
+    - function,
+    - how to use,
+    - tutorial,
+    - etc.
+3) Javascript 👀
+\
+`Javascript` atau `JS` merupakan _scripting language_ yang digunakan dalam web development, web applications, game development, dan masih banyak lagi. Dan perlu diketahui untuk mengembangkan sebuah fitur dinamis dalam sebuah web tidak dapat tergantung hanya pada `HTML` dan `CSS` saja, tetapi justru kedinamisan fitur tersebut terjadi dengan menggunakan `JS` ini sesuai dengan tujuan fiturnya.
```

Comment:
![DeepinScreenshot_select-area_20210810143509](https://user-images.githubusercontent.com/18365721/128827194-d25ee9fe-39c5-4fd6-afb2-b786fc82749e.png)

Please check again the readme file as this one doesnt seem to render as what you would like it to be, use the appropriate readme viewer in vscode if you want to make it easier to see it

Reply:
okay bang, i'll be check again and resolve this problem soon

Reply:
Would you mind if i fix your grammar a little bit @adhiiimf ? just to make things clearer you know wkwk

Reply:
cause it should've been `i'll check it again` instead of `i'll be check again`

Reply:
wkwkwk thankyou to correct my english grammar sir

Reply:
wkwk and also this one should've been `for correcting` instead of `to correct`, yeah its painful to do this grammar stuff but it'll be worth the price i promise wkwk